index.js

```diff
@@ -1,32 +1,29 @@
 /* eslint-env node */
 'use strict';
 
-var path = require('path');
-var filterInitializers = require('fastboot-filter-initializers');
-var VersionChecker = require('ember-cli-version-checker');
-var mergeTrees = require('broccoli-merge-trees');
+const path = require('path');
+const VersionChecker = require('ember-cli-version-checker');
+const mergeTrees = require('broccoli-merge-trees');
 
 module.exports = {
   name: 'ember-cli-head',
 
-  treeForApp: function(defaultTree) {
-    var checker = new VersionChecker(this);
-    var emberVersion = checker.for('ember-source', 'npm');
+  treeForApp(defaultTree) {
+    let checker = new VersionChecker(this);
+    let emberVersion = checker.for('ember-source', 'npm');
 
     if (!emberVersion.version) {
       emberVersion = checker.for('ember', 'bower');
     }
 
-    var trees = [defaultTree];
+    let trees = [defaultTree];
 
     // 2.9.0-beta.1 - 2.9.0-beta.5 used glimmer2 (but 2.9.0 did not)
     // 2.10.0-beta.1+ includes glimmer2
     if (!(emberVersion.gt('2.9.0-beta') && emberVersion.lt('2.9.0')) && !emberVersion.gt('2.10.0-beta')) {
```

Comment:
My PR actually exposes an API for these kind of apps. `addToFastBootTree`. pre-fastboot 1.0 this hook will not be called but post can be called.